server/bleep/src/webserver/studio.rs

```diff
@@ -53,14 +53,50 @@ pub struct ListItem {
     id: String,
     name: String,
     modified_at: NaiveDateTime,
+    repos: Vec<String>,
+    most_common_ext: String,
 }
-
 pub async fn list(app: Extension<Application>) -> webserver::Result<Json<Vec<ListItem>>> {
-    sqlx::query_as! { ListItem, "SELECT id, name, modified_at FROM studios" }
+    let studios = sqlx::query!("SELECT id, name, modified_at FROM studios")
         .fetch_all(&*app.sql)
         .await
-        .map_err(Error::internal)
-        .map(Json)
+        .map_err(Error::internal)?;
+
+    let mut list_items = Vec::new();
+
+    for studio in studios {
+        let context_json: String = sqlx::query!("SELECT context FROM studios WHERE id = ?", studio.id)
```

Comment:
I wanted to avoid this additional sqlx call for each studio, but putting context into select on line 60 just returns an error
```
error: failed to find data for query SELECT id, name, modified_at, context FROM studios
  --> server/bleep/src/webserver/studio.rs:68:35
   |
68 |       let studios: Vec<StudioRow> = sqlx::query_as!(
   |  ___________________________________^
69 | |         StudioRow,
70 | |         "SELECT id, name, modified_at, context FROM studios"
71 | |     )
   | |_____^
   |
   = note: this error originates in the macro `$crate::sqlx_macros::expand_query` which comes from the expansion of the macro `sqlx::query_as` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
We should swap this back to a single SQL call. The reason it was giving that error is that you have to use the `sqlx` CLI to update the query info JSON. `sqlx` actually loads the database and examines the query in order to do this.